presto-native-execution/scripts/dockerfiles/prestissimo-runtime.dockerfile

```
@@ -39,6 +39,15 @@ FROM ${BASE_IMAGE}
 ENV BUILD_BASE_DIR=_build
 ENV BUILD_DIR=""
 
+# TODO: This is a temporary fix for issue: https://github.com/prestodb/presto/issues/25531
+# When there is a proper fix, update this piece of code.
+RUN apt-get update && \
```

Hadolint (2.12.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)